packages/adaptive_scaffold/example/pubspec.yaml

```diff
@@ -15,10 +13,30 @@ dependencies:
   flutter:
     sdk: flutter
 
+
+
+  cupertino_icons: ^1.0.2
+
```

Comment:
Is this a dependency that the app uses? Because I remember seeing it come up as a default.